samples/spring-customer-registry-views-quickstart/src/it/java/customer/api/CustomerIntegrationTest.java

```diff
@@ -2,34 +2,29 @@
 
 
 import customer.Main;
-import kalix.springsdk.KalixConfigurationTest;
+import kalix.springsdk.KalixIntegrationTestKitSupport;
 import org.hamcrest.core.IsEqual;
-import org.junit.BeforeClass;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 import org.junit.runner.RunWith;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.context.annotation.Import;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.test.context.TestPropertySource;
 import org.springframework.test.context.junit4.SpringRunner;
 import org.springframework.web.reactive.function.client.WebClient;
 import reactor.core.publisher.Mono;
 
 import java.time.Duration;
-import static java.time.temporal.ChronoUnit.SECONDS;
-import java.util.concurrent.TimeUnit;
 import java.util.UUID;
+import java.util.concurrent.TimeUnit;
 
+import static java.time.temporal.ChronoUnit.SECONDS;
 import static org.awaitility.Awaitility.await;
 
 @RunWith(SpringRunner.class)
 @SpringBootTest(classes = Main.class)
-@Import(KalixConfigurationTest.class)
-@TestPropertySource(properties = "spring.main.allow-bean-definition-overriding=true")
-public class CustomerIntegrationTest {
+public class CustomerIntegrationTest extends KalixIntegrationTestKitSupport {
```

Comment:
At the end, the path with less burden to the user is to extends this class. There are a few configs in place that we don't want to ask users to copy over and over. Easier to say, extends this and everything will work (hopefully).